client/state/reader/streams/selectors/get-reader-stream-should-request-recommendations.js

```diff
@@ -2,8 +2,10 @@
  * Internal dependencies
  */
 import { getDistanceBetweenRecs, RECS_PER_BLOCK } from 'reader/stream/utils';
-import getReaderFollows from 'state/selectors/get-reader-follows';
-import getReaderStream from 'state/selectors/get-reader-stream';
+import { getReaderFollows } from 'state/reader/follows/selectors';
+import getReaderStream from './get-reader-stream';
```

Comment:
Should this import from `state/reader/streams/selectors` to be more explicit?

This can be seen multiple times in this PR.